src/pages/detail-page/subpages/tab-panels/MetadataInformationPanel.tsx

```diff
@@ -1,7 +1,48 @@
-import React from "react";
+import React, { useEffect, useMemo, useState } from "react";
+import { useDetailPageContext } from "../../context/detail-page-context";
+import NavigatablePanel from "../components/NavigatablePanel";
+import ContactBlock from "../components/ContactBlock";
 
 const MetadataInformationPanel = () => {
-  return <div>MetadataInformationCard</div>;
+  const context = useDetailPageContext();
+  const metadataContact = context.collection
+    ?.getContacts()
+    ?.filter((contact) => contact.roles.includes("metadata"));
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    if (!metadataContact) {
+      setIsLoading(true);
+    } else {
+      setIsLoading(false);
+    }
+  }, [metadataContact]);
```

Comment:
if you use suggested change for metadataContacts, this can become:

```javascript
useEffect(() => {
  setIsLoading(metadataContacts.length === 0);
}, [metadataContacts]);
```

which prevents toggling between true and false unnecessarily, causing fewer re-renders.